smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/middlewares/OperationInputBodyMiddleware.kt

```diff
@@ -27,9 +30,15 @@ class OperationInputBodyMiddleware(
         operationStackName: String,
     ) {
         val inputShapeName = MiddlewareShapeUtils.inputSymbol(symbolProvider, model, op).name
+        val outputShapeName = MiddlewareShapeUtils.outputSymbol(symbolProvider, model, op).name
+        val errorShapeName = MiddlewareShapeUtils.outputErrorSymbolName(op)
         val hasHttpBody = MiddlewareShapeUtils.hasHttpBody(model, op)
         if (hasHttpBody || shouldRender) {
-            writer.write("$operationStackName.${middlewareStep.stringValue()}.intercept(position: ${position.stringValue()}, middleware: ${inputShapeName}BodyMiddleware())")
+            val requestBindings = httpBindingResolver.requestBindings(op)
+            val alwaysSendBodyParam = if (shouldRender) "alwaysSendBody: true" else ""
```

Comment:
`shouldRender` was being used as a way to dictate whether or not we should render the call to intercept, and in this proposed change, we seem to be conflating it with whether or not there is a body param.  Can you think of a cleaner way to write this?

Reply:
resolving!